### PR TITLE
Welcome blog post for freshers

### PR DESCRIPTION
**Pull request by sbrl**
This PR contains an initial draft of a post welcoming freshers to Freeside. If all goes well, it should go out just before the first week of this next semester.

Feel free to read and review it!

### CODE REVIEW

**Comment by sbrl** [comment]
We could arrange a meet-and-greet and put that in this blog post too.

File: _posts/2018-10-01-welcome-to-freeside.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+title: Welcome 2018 Freshers to Freeside!
+author: Starbeamrainbowlabs
+tags:
+- updates
+---
+
+This is an introductory post to Freeside, a loose group of students who use Linux and want to educate themselves (and others!) about alternative open computing.
+
+## How to find us
+Our physical space is a new Lab that has been purpose built - the _Ritche Lab_ - which is located next to LTD on the 3rd floor of the Robert Blackburn building. It contains computers running Fedora Linux for all students to use - simply register an account [here](https://profiles.freeside.co.uk/join) to get your login credentials.
```
Comment (Epictek) [inline _posts/2018-10-01-welcome-to-freeside.md]: link to https://freeside.co.uk/2018/04/15/freeside-is-now-open for more info about the room/computers?

**Comment by Epictek** [comment]
@sbrl for now just add a "keep an eye on our social media for upcoming meet and greet events"?

**Comment by Epictek** [comment]
Another idea would be "Who we are looking for/Who can help" section? Examples?:
* Experienced Linux users.
* Experieced Linux Admins.
* Developers (Link to github projects).
* Graphic designers
* Anyone intrested in free software!
* Anyone else that has any other skills

**Comment by sbrl** [comment]
Ah, good idea. Here's the start of a new section for this blog post:

```markdown
## Help out!
We're always looking for more people to help out. Whether you're an experienced Linux user or just starting out, there are lots of ways that you can contribute!
```

....I've kind of stalled here though. Not sure how to continue this new paragraph :confused:

**Comment by Epictek** [comment]
Not sure how to complete that either, Maybe the blog post is fine how it is? :)

**Comment by sbrl** [comment]
Coming back to this a few days later seems to have helped. What about something like this?

```
## Help out!
We're always looking for more people to help out. Whether you're an experienced Linux user or just starting out, there are lots of ways that you can contribute! Check out [our forums](https://forums.freeside.co.uk) for a list of things that need doing - you're welcome to propose new ideas too!
```

**Comment by Epictek** [comment]
Great!

**Comment by Epictek** [comment]
LGTM

**Pull request merged**